docs/network/fees.md

```diff
@@ -2,7 +2,11 @@
 title: Fee structure
 ---
 
-import GasPriceWarning from '@site/docs/conrefs/gas-price-warning.md';
+The Etherlink gas price (and therefore the fee for a given transaction) varies based on the activity on the chain.
+As activity increases, fees increase, and vice versa.
+For information about estimating fees, see [Estimating fees](/building-on-etherlink/estimating-fees).
+
+Etherlink fees include the cost of running the transaction and writing the transaction to layer 1 but not a voluntary tip.
```

Comment:
```suggestion
Etherlink fees include the cost of running the transaction and writing the transaction to layer 1. It's not possible to set a voluntary tip or priority fee to increase the chances of a transaction being accepted, if included as part of the transaction parameters it will be ignored.
```